docs/monitoring/index.md

```diff
@@ -54,6 +54,12 @@ Read more in the [Reports and Components](./reports-and-components) guide.
 
 ![Sematext Cloud Monitoring Guide - Monitoring Report Menu](https://sematext.com/docs/images/guide/monitoring/sematext-monitoring-guide-report-menu.png) 
 
+
+### Quickly start to monitor with Autodiscovery
+
+The only thing you'll have to install is Sematext Agent. After that you can enable/disable monitoring of each discovered service type from the user interface. To learn more visit [Autodiscovery Monitoring](./autodiscovery). Of course, if you wish to have full control over what is monitored and avoid configuring things in the user interface, you can still do that - application will show you manual monitoring installation instructions too.
```

Comment:
After that you => After that, you
Autodiscovery Monitoring => Service Autodiscovery
application will show you -- which application?  Rephrase...
too => , too